### PR TITLE
fix: 🐛 Surah icon order after sorting in Revelation Order tab

### PR DESCRIPTION
**Pull request by ThatSameer**
Bismillah

### Issue
#1957 

### Summary
The Surah icons were not updating when changing the sorting order in the 'Revelation Order' tab. The incorrect index was being passed into the `SurahPreviewRow` component which is responsible for getting the chapter icon.
This PR fixes the issue by using the correct value for the chapter id.

### Unrelated changes
I have added additional config in the `settings.json` file which is being used by the [i18n Ally VS Code extension](https://marketplace.visualstudio.com/items?itemName=lokalise.i18n-ally). This extension makes using i18n easier.
If you would like me to remove this config from this PR, please let me know.

### Test Plan
1. Go to [localhost](http://localhost:3000/) and click on the 'Revelation Order' tab
2. Check the Arabic chapter icons are mapping correctly to the Surah
3. Change the sort by order to descending and again check if the icons have updated to the correct name

### Screenshots

BEFORE
image

AFTER
image

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
+    "src/styles/themes/_light.scss" // include one of the theme file
+  ],
+  "prettier.disableLanguages": [
+    "javascript",
+    "javascriptreact",
+    "typescript",
+    "typescriptreact",
+    "scss"
+  ],
+  "i18n-ally.localesPaths": [
```
Comment (ahmedriad1) [inline .vscode/settings.json:26]: Salam alykum Sameer,

JAK for your contribution!

The i18n Ally extension looks cool, thanks for including it! But can you please remove it from this PR and make a new one for it as it's out of scope?

File: src/components/chapters/RevelationOrderView.tsx [hunk 2]
```diff
@@ -84,7 +84,7 @@ const RevelationOrderView = ({ isDescending, chapters }: RevelationOrderViewProp
           onKeyPress={() => onSurahClicked(chapter.id)}
         >
           <SurahPreviewRow
-            chapterId={REVELATION_ORDER[revelationOrderIndex]}
+            chapterId={+chapter.id}
```
Comment (ahmedriad1) [inline src/components/chapters/RevelationOrderView.tsx:87]: nit: change `+chapter.id` to `Number(chapter.id)`

File: .vscode/settings.json (hunk 1 above)
Comment (ThatSameer) [reply .vscode/settings.json:26]: Walaikumsalaam.
Jzk, have updated in recent commit

File: src/components/chapters/RevelationOrderView.tsx (hunk 2 above)
Comment (ThatSameer) [reply src/components/chapters/RevelationOrderView.tsx]: Updated 👍

**Review by osamasayed — APPROVED** [review]

**Pull request merged**